src/main/java/http/request/Request.java

```diff
@@ -1,5 +1,41 @@
 package http.request;
 
+import java.util.*;
```

Comment:
👏 short and clear, this should work very well as Request's abstraction

Reply:
Thank you 👍